lib/bundle/brew_dumper.rb

```diff
@@ -69,18 +73,18 @@ def formula_inspector f
       args << "devel" if keg["version"].gsub(/_\d+$/, "") == f["versions"]["devel"]
       args.uniq!
       {
-        name: f["name"],
-        full_name: f["full_name"],
-        args: args,
-        version: keg["version"],
-        dependencies: f["dependencies"],
-        requirements: f["requirements"],
+        :name => f["name"],
+        :full_name => f["full_name"],
+        :args => args,
+        :version => keg["version"],
+        :dependencies => f["dependencies"],
+        :requirements => f["requirements"],
```

Comment:
This is changed by Rubocop(Using `brew style --fix <path to homebrew/bundle>`). Maybe we can change the rubocop rule if you want.